src/swsssdk/scripts/sonic-db-cli

```diff
@@ -1,11 +1,24 @@
 #!/usr/bin/python
+from __future__ import print_function
```

Comment:
just want to make sure if this PR rebased to the latest, I saw there is another commit yesterday with this changes, it should not be shown in your changes set.

Reply:
Ok, I too notice this, but I did a rebase and it was clean, not sure why it is showing it. Let me check this

Reply:
looks to me that there is no merge conflict, hence resolving it. Will make sure that merging is fine.